pkg/resource/certificate_authority/sdk.go

```diff
@@ -745,82 +740,8 @@ func (rm *resourceManager) sdkUpdate(
 	desired *resource,
 	latest *resource,
 	delta *ackcompare.Delta,
-) (updated *resource, err error) {
```

Comment:
Mmmm was this intentional? looks like all CA update code is gone.

Reply:
And the e2e tests are passing.. which means that they are missing the case of CA updates

Reply:
We don't want the user to be able to update the CA resource, the update happens through the CAActivation resource

Reply:
Not true - we want the CA resource to be updated via the UpdateCertificateAuthority API, which will be called as part of the CA resource.

The same API will also be called as part of the CAA resource, but just to toggle the status of the CA.